doc/src/content/xdocs/spec.xml

```diff
@@ -564,7 +565,7 @@
                 followed by the serialized string:
                 <source>02 02 61</source></li>
             </ul>
-            <p><em>NOTE</em>: Currently for C/C++ implementtions, the positions are practically an int, but theoretically a long. 
+            <p><em>NOTE</em>: Currently for C/C++ implementtions, the positions are practically an int, but theoretically a long.
```

Comment:
```suggestion
            <p><em>NOTE</em>: Currently for C/C++ implementations, the positions are practically an int, but theoretically a long.
```
While you're at this sentence :D

Reply:
Sounds good. I'll add this change to the commit!